### PR TITLE
Updated links on line 820 and 828 in blogs.yml

### PR DESCRIPTION
**Pull request by 7daytheory**
Updated the 2 links that needed to be updated.

Sorry for a bit of a mess (4 commits) - my editor 'prettified' the double quotes so I went back and undid that and then I added the new link that was given to me just now.

7Daytheory

### CODE REVIEW

**Comment by 7daytheory** [comment]
Sure thing! Appreciate working with me while I start with working on open sources projects. Will let u know when updated

**Comment by 7daytheory** [comment]
> Hey, @7daytheory, thanks for the update!
> 
> But, could you please update the lines **852/858**?
> For example:
> 
> ```
>     title: "Setting Cypress and Cucumber in e2e tests"
>     url: https://medium.com/assertqualityassurance/setting-cypress-and-cucumber-in-e2e-tests-6239adac1077
>     sourceName: Agile Testers
>     sourceUrl: https://agiletesters.github.io/
>     author: Lucas Fraga
>     date: Jan 24, 2019
>     img: /img/examples/cypress-com-cucumber.jpg
> ```
> 
> Thanks again!

Where do I get the image for line 858 ?

**Comment by 7daytheory** [comment]
> Hey, @7daytheory, thanks for the update!
> 
> But, could you please update the lines **852/858**?
> For example:
> 
> ```
>     title: "Setting Cypress and Cucumber in e2e tests"
>     url: https://medium.com/assertqualityassurance/setting-cypress-and-cucumber-in-e2e-tests-6239adac1077
>     sourceName: Agile Testers
>     sourceUrl: https://agiletesters.github.io/
>     author: Lucas Fraga
>     date: Jan 24, 2019
>     img: /img/examples/cypress-com-cucumber.jpg
> ```
> 
> Thanks again!

Would this be correct ?

  - title: "Setting Cypress and Cucumber in e2e tests"
    url: https://medium.com/assertqualityassurance/setting-cypress-and-cucumber-in-e2e-tests-6239adac1077
    sourceName: Medium
    sourceUrl: https://www.medium.com
    author: Lucas Fraga
    date: April 3, 2019
    img: /img/examples/cypress-com-cucumber.jpg

**Comment by uLucasFraga** [comment]
@7daytheory can be, no problem! 
thanks :)

**Comment by uLucasFraga** [comment]
> Would this be correct ?
> 
> * title: "Setting Cypress and Cucumber in e2e tests"
>   url: https://medium.com/assertqualityassurance/setting-cypress-and-cucumber-in-e2e-tests-6239adac1077
>   sourceName: Medium
>   sourceUrl: https://www.medium.com
>   author: Lucas Fraga
>   date: April 3, 2019
>   img: /img/examples/cypress-com-cucumber.jpg

@7daytheory can be, no problem!
thanks :)

**Comment by 7daytheory** [comment]
> @7daytheory There's a bunch of extra lines of code in the blogs.yml that shouldn't be there - like it's adding content from GitHub. This will need to be removed before merging in. https://github.com/cypress-io/cypress-documentation/pull/3020/files

Sorry - copied and pasted code from github because my editor prettified the code. Didn't take it as seriously as I should of bc the act itself was easy. Learning from this mistake for future - as this was my first contribution

File: source/_data/blogs.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# small links with title and url
+ # small links with title and url
```
Comment (jennifer-shehane) [inline source/_data/blogs.yml:1]:
```suggestion
# small links with title and url
```

**Comment by jennifer-shehane** [comment]
@7daytheory Thanks for the contribution!

**Pull request merged**